src/SessionOne.java

```diff
@@ -5,9 +5,9 @@ public static void main(String[] args) {
         Scanner input = new Scanner(System.in);
         System.out.println("Press 1, if you want to see if it's an SessionOne number");
         System.out.println("Press 2, whether you want to see if it's an Strong number");
+        System.out.println("Press 3, whether you want to see if it's an Perfect number");
```

Comment:
I don't know, what's happening even this comment disappeared the  first time.. I was saying that the master branch has your commits, but I was (i don't know why) expecting those to be merged once the PR for the feature/practice branch is ok.. I can see that's not the case.. And also, I don't know how to see the files within each branch... I was trying that yesterday to see the thing with the .idea folder, but I was not able to find them. Help meeeee please

Reply:
First PR finally MERGED!!! 🚀 How does it feel, cool right? Great job! 💪 

Uh disappearing comments, huh! 👻 I remember yesterday Github was having issues, at least in The Netherlands, don't know about two days ago in Cedritos. Or maybe you just forgot to submit the comment.  Also, don't reply by editing the comments because it makes difficult to keep track of the conversation, I just randomly find out about it, you won't get a notification about comments edits, if the tool is really not working, just send me an email or something.

Getting to the point, PRs are a living thing, as you may have noticed already, if you update one of the branches of a PR, then the PR itself will get updated, so for example after merging your beautiful PR, `master` got updated, then this PR got updated as well, since you want to merge the maven configuration branch into `master`. Then you might have to update your PR accordingly depending on the case. Right now this PR has conflicts that should be solved, take care of those.

How to check branches in here? 🤔 

### One way:

1. Go to the code's tabe:
<img width="797" alt="Screenshot 2020-02-28 at 10 30 21" src="https://user-images.githubusercontent.com/10931891/75535364-727f9b80-5a15-11ea-9523-c545f38d5bdb.png">

2. Once there, use the branches dropdown to choose the branch you want to check:
<img width="435" alt="Screenshot 2020-02-28 at 10 35 55" src="https://user-images.githubusercontent.com/10931891/75536774-22550900-5a16-11ea-81fc-6ed32a5d616b.png">

3. Then you will be able to check only the code for that specific branch, for example:
<img width="656" alt="Screenshot 2020-02-28 at 10 39 29" src="https://user-images.githubusercontent.com/10931891/75537335-1f0e4d00-5a17-11ea-86d1-9a37e59bd314.png">

### Another way:

That you can use from the PR detail screen:

1. Go the PR's page:
<img width="610" alt="Screenshot 2020-02-28 at 10 46 37" src="https://user-images.githubusercontent.com/10931891/75537747-dacf7c80-5a17-11ea-8cf4-5c7acd7e5be6.png">

2. Click on the name of the branch you want to check:
<img width="345" alt="Screenshot 2020-02-28 at 10 51 24" src="https://user-images.githubusercontent.com/10931891/75538045-487ba880-5a18-11ea-8274-ddf458b0360e.png">
What if you click that clipboard icon next to the branch's name? What does it do? One might wonder. Are you brave enough to click it? :eyes: (For the nth time, explore your tool)

3. There you are, now you can review your branch's code:
<img width="689" alt="Screenshot 2020-02-28 at 10 52 27" src="https://user-images.githubusercontent.com/10931891/75538190-84167280-5a18-11ea-82b2-fc40f536fdce.png">

Reply:
> 👀 This doesn't belong to this branch, I am guessing that you created this branch from `feature/practice` instead of `master` that's why you don't see my latest commits. To keep in mind:
> 
> * Feature branches are often created from master, not from another feature branch. ... **_That was the same thing that I did wrong in the Google project_**
> * It doesn't have to be like this all the time:
>   
>   * Hypothetical scenario: You are working with two developers in building a feature, hence you all create a feature branch `feature/our-cool-feature` each one of you it's going to contribute with different tasks for this particular feature, then you create branches from the `feature/our-cool-feature` branch, those will eventually be merged on the base feature branch, which will be merged later into `master`.
>   * You work in an environment where the base branch for development is not `master` but for example `develop`.
> 
> Taking this into account, try to be mindful of what you are doing, think where are you creating your branches from, and why. Before you continue, do the following exercise:
> 
> * Look at this branch commits: https://github.com/c-machado/SimpliLearnSecond/commits/feature/maven-config
> * Look at `feature/practice` commits: https://github.com/c-machado/SimpliLearnSecond/commits/feature/practice
> * Look at `master` commits: https://github.com/c-machado/SimpliLearnSecond/commits/master
> * Compare them carefully, what do you notice? Can you spot the differences between them? makes more sense now, why do you see some things that you shouldn't see, and why you don't see things that you were expecting to see?

Well, as far as I can see all the commits that I created during all the time I worked in the branch feature/practice are now in Master, and also I could see that some of the commits of the branch feature/practice are in feature/maven-config, I guess it was the commits in there until the moment I created this branch, because I wrongly used that one as base of this. But, now I have a question, what if I had created feature/maven-config from Master, now I would be able to see all the commits in Master here? :bowtie:

Reply:
> First PR finally MERGED!!! 🚀 How does it feel, cool right? Great job! 💪
> 
> Uh disappearing comments, huh! 👻 I remember yesterday Github was having issues, at least in The Netherlands, don't know about two days ago in Cedritos. Or maybe you just forgot to submit the comment. Also, don't reply by editing the comments because it makes difficult to keep track of the conversation, I just randomly find out about it, you won't get a notification about comments edits, if the tool is really not working, just send me an email or something.
> 
> Getting to the point, PRs are a living thing, as you may have noticed already, if you update one of the branches of a PR, then the PR itself will get updated, so for example after merging your beautiful PR, `master` got updated, then this PR got updated as well, since you want to merge the maven configuration branch into `master`. Then you might have to update your PR accordingly depending on the case. Right now this PR has conflicts that should be solved, take care of those.
> 
> How to check branches in here? 🤔
> 
> ### One way:
> 1. Go to the code's tab:
> 
> <img alt="Screenshot 2020-02-28 at 10 30 21" width="797" src="https://user-images.githubusercontent.com/10931891/75535364-727f9b80-5a15-11ea-9523-c545f38d5bdb.png">
> 
> 1. Once there, use the branches dropdown to choose the branch you want to check:
> 
> <img alt="Screenshot 2020-02-28 at 10 35 55" width="435" src="https://user-images.githubusercontent.com/10931891/75536774-22550900-5a16-11ea-81fc-6ed32a5d616b.png">
> 
> 1. Then you will be able to check only the code for that specific branch, for example:
> 
> <img alt="Screenshot 2020-02-28 at 10 39 29" width="656" src="https://user-images.githubusercontent.com/10931891/75537335-1f0e4d00-5a17-11ea-86d1-9a37e59bd314.png">
> 
> ### Another way:
> That you can use from the PR detail screen:
> 
> 1. Go the PR's page:
> 
> <img alt="Screenshot 2020-02-28 at 10 46 37" width="610" src="https://user-images.githubusercontent.com/10931891/75537747-dacf7c80-5a17-11ea-8cf4-5c7acd7e5be6.png">
> 
> 1. Click on the name of the branch you want to check:
> 
> <img alt="Screenshot 2020-02-28 at 10 51 24" width="345" src="https://user-images.githubusercontent.com/10931891/75538045-487ba880-5a18-11ea-8274-ddf458b0360e.png">
> 
> What if you click that clipboard icon next to the branch's name? What does it do? One might wonder. Are you brave enough to click it? 👀 (For the nth time, explore your tool)
> 
> 1. There you are, now you can review your branch's code:
> 
> <img alt="Screenshot 2020-02-28 at 10 52 27" width="689" src="https://user-images.githubusercontent.com/10931891/75538190-84167280-5a18-11ea-82b2-fc40f536fdce.png">

Actually yes, it felt awesome 😌. Thanks for all the work that you did here. 

QQ, what exactly is 'My-branch' in the list of branches?

That clipboard works for copying the branch name :neckbeard:

Reply:
My branch is a branch I created to check something, but it can be deleted. I will delete it.

Reply:
I have done it, but in a very basic way I guess ... baby steps :)

Reply:
Fair enough, I've been too demanding regarding that. Discover it at your own pace, try to stay curious.

Reply:
No, it's not that. I realized I need to be more curious, but my mind it's like.. that's easy.. whatever.. but it's not because I know how to do something, it's just a bad habit. Or I think the questions are too obvious and it's like 'no, you can't ask, that'... or so on..

Reply:
Ok then, try to stay open, don't be ashamed to ask even when questions might seem obvious. Nobody will judge (at least not in here) And if somebody does, remember that with any tiny piece of knowledge that you get, you outgrow the person that you were yesterday. That should be the mindset.

Reply:
fair enough :) I know, you won't judge me... still I don't know why is so hard to ask. But, I'm going to do it, from now on. Actually, I was checking my CR with Felipe yesterday, and I asked I lot of questions, that helps me to understand a lot of things, so, I'm going to start doing it as much as I can.